src/twisted/python/test/test_release.py

```diff
@@ -1400,7 +1412,7 @@ def test_remove(self):
         """
         reposDir = self.makeRepository(self.tmpDir)
         testFile = reposDir.child('some-file')
-        testFile.setContent("something")
```

Comment:
I don't think a module can be considered ported as long as it uses `str` with `FilePath.child`.  Some Python APIs legitimately take `str` as an argument - but not this one.